lib/services/firebase_service.dart

```diff
@@ -198,10 +198,11 @@ class FirebaseService extends IFirebaseService {
     return signature;
   }
 
-  Future<void> addDataToFormsCollection(
+  Future<String> addDataToFormsCollection(
       {@required Map<String, dynamic> data,
       @required String formName,
-      @required String hn}) async {
+      @required String hn,
+      String formTime}) async {
```

Comment:
What is form time? Is it the time that the form is created ?

Reply:
Not form creation time but it is form time of vital sign form => 02.00 06.00 10.00 14.00 18.00 22.00 (every 4 hour) I think it make more easy when use the data to plot graph in dashboard because sometimes the medical team might not be able to save the vital sign form on time.